sdk/storage/azblob/container/examples_test.go

```diff
@@ -267,7 +267,7 @@ func Example_container_ClientListBlobsHierarchy() {
 			log.Fatal(err)
 		}
 		for _, blob := range resp.ListBlobsHierarchySegmentResponse.Segment.BlobItems {
-			fmt.Println(*blob.Name)
+			fmt.Println(*blob.Name.Content)
```

Comment:
This will be a breaking change for the customers as the response values are getting changed here.

Reply:
The work around is to:

1. Use a swagger readme transform to rename this type and make it internal.
2. Create a new hand-written type, with the same properties as before
3. Create a hand-written function that can be used to convert the new hand-written class into this generated class that is accepted as a parameter by the generated code.